src/__tests__/fireEvent.test.js

```diff
@@ -242,3 +282,25 @@ test('is not fooled by non-native disabled prop', () => {
   fireEvent.press(screen.getByText('Trigger Test'));
   expect(handlePress).toHaveBeenCalledTimes(1);
 });
+
+const TextInputWrapper = ({ placeholder, onChangeText }) => {
+  return <TextInput placeholder={placeholder} onChangeText={onChangeText} />;
+};
+
+test('should not be fooled by non-native editable prop', () => {
```

Comment:
Hm I added this test since we have a similar one for TouchableOpacity above it, but it seems like it is not necessary in the case of TextInput where we don't care about the event propagating to the custom parent component. I'll remove it thanks!